assets/sass/_helper.scss

```diff
@@ -0,0 +1,17 @@
+/*
+ * Hide only visually, but have it available for screen readers:
+ * http://snook.ca/archives/html_and_css/hiding-content-for-accessibility
+ */
+
+// Added as a class because this should to be obvious in HTML
+
+.is-visuallyhidden {
```

Comment:
Yeah makes sense. @jonathanconway's PR #213 has that naming in it too :)
Good pickup